src/test/java/org/folio/rest/impl/FeeFineReportsAPITest.java

```diff
@@ -192,11 +193,15 @@ public void emptyReportWhenRefundedAfterEndDate() {
 
   @Test
   public void shouldReturn422WhenRequestIsNotValid() {
-    refundReportsClient.getFeeFineRefundReports("2020-01-01", null, HTTP_UNPROCESSABLE_ENTITY);
     refundReportsClient.getFeeFineRefundReports(null, "2020-01-01", HTTP_UNPROCESSABLE_ENTITY);
     refundReportsClient.getFeeFineRefundReports("not-a-date", "2020-01-01", HTTP_UNPROCESSABLE_ENTITY);
-    refundReportsClient.getFeeFineRefundReports("2020-01-01", "not-a-date", HTTP_UNPROCESSABLE_ENTITY);
-    refundReportsClient.getFeeFineRefundReports(null, null, HTTP_UNPROCESSABLE_ENTITY);
+  }
+
+  @Test
+  public void shouldReturn200WhenRequestIsValid() {
+    refundReportsClient.getFeeFineRefundReports("2020-01-01", null, HTTP_OK);
+    refundReportsClient.getFeeFineRefundReports("2020-01-01", "not-a-date", HTTP_OK);
```

Comment:
This doesn't seem right to me. I don't know if we should just ignore the incorrect date as if we didn't receive it at all.